readme/Rewrites.scalatex

```diff
@@ -139,6 +139,36 @@
       Such members already have a return type of @code{Unit} and sometimes this is unexpected.
       Adding an explicit return type makes it more obvious.
 
+  @sect(DottyVarArgPattern.toString)
+    @p
+      Replaces @code{@} symbols in VarArg patterns with a colon (@code{:}). See @lnk{http://dotty.epfl.ch/docs/reference/changed/vararg-patterns.html}
```

Comment:
This line here has a syntax error, I think the `@` symbol needs to be escaped with `@@`. To test locally run `readme/run`.